experiment/autobumper/bumper/bumper.go

```diff
@@ -84,55 +84,55 @@ func (af *fileArrayFlag) Set(value string) error {
 	UpstreamURLBase string `yaml:"upstreamURLBase"`
-	//The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.
+	// The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.
 	IncludedConfigPaths []string `yaml:"includedConfigPaths"`
-	//The config paths to be excluded in this bump, in which only .yaml files will be considered.
+	// The config paths to be excluded in this bump, in which only .yaml files will be considered.
 	ExcludedConfigPaths []string `yaml:"excludedConfigPaths"`
-	//The extra non-yaml file to be considered in this bump.
+	// The extra non-yaml file to be considered in this bump.
 	ExtraFiles []string `yaml:"extraFiles"`
-	//The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.
+	// The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.
 	TargetVersion string `yaml:"targetVersion"`
-	//The name used in the address when creating remote. Format will be git@github.com:{GitLogin}/{RemoteName}.git
+	// The name used in the address when creating remote. Format will be git@github.com:{GitLogin}/{RemoteName}.git
 	RemoteName string `yaml:"remoteName"`
-	//The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
+	// The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
 	HeadBranchName string `yaml:"headBranchName"`
-	//List of prefixes that the autobumped is looking for, and other information needed to bump them
+	// List of prefixes that the autobumped is looking for, and other information needed to bump them. Must have at least 1 prefix.
 	Prefixes []Prefix `yaml:"prefixes"`
 }
 
 // Prefix is the information needed for each prefix being bumped.
 type Prefix struct {
-	//Name of the tool being bumped
+	// Name of the tool being bumped
 	Name string `yaml:"name"`
-	//The image prefix that the autobumper should look for
+	// The image prefix that the autobumper should look for
 	Prefix string `yaml:"prefix"`
-	//File that is looked at when bumping to match upstream
+	// File that is looked at to determine current upstream image when bumping to upstream. Required only if targetVersion is "upstream"
 	RefConfigFile string `yaml:"refConfigFile"`
-	//File that is looked at when bumping to match upstreamStaging
+	// File that is looked at to determine current upstream staging image when bumping to upstream staging. Required only if targetVersion is "upstream-staging"
 	StagingRefConfigFile string `yaml:"stagingRefConfigFile"`
-	//Repo used when generating pull request
+	// Repo that the PR for the bump will be pushed to. This is used to generate comparison link in the PR summary.
```

Comment:
"Repo that the PR for the bump will be pushed to." is a bit confusing to me since we push the PR branch to the fork of the target repo, but the PR is created in the target repo itself. 
Furthermore, the comparison link that is used in the PR summary should actually come from the upstream repo where the images are sourced in order to actually show the changes made between the old version and the version to which we are bumping.

So if we're bumping Prow images in `GoogleCloudPlatform/oss-test-infra` (the target repo), the fork would be something like `bot-name/oss-test-infra` and the source repo of the images to use for comparison links would be `kubernetes/test-infra`. Which of these repos is this field intended to represent? Do we need more fields to distinguish these?

Reply:
Oh! This makes much more sense now. I was only looking at examples where the target and source repo were the same and so I was confused as to what the distinction was. This is meant to refer to the source repo. Thank you for the explanation